pytest_stf/plugin.py

```diff
@@ -103,16 +103,19 @@ def fixture_allocated_phone(pytestconfig, lockable):
     assert requirements, 'phone_requirements required'
     assert 'platform' in requirements, 'platform required'
 
+    allocation_timeout = pytestconfig.getoption('allocation_timeout') or 0
```

Comment:
default value in appium-stf-client is 60s: https://github.com/OpenTMI/stf-appium-python-client/blob/6d568611a7167eaa3629dffab006b5799e025859/stf_appium_client/StfClient.py#L242 . perhaps here should be the same. py-lockable default timeout seems to be longer, 1000s: https://github.com/jupe/py-lockable/blob/c62662a8a5b9410c7abe5622b3e3513f15c6e414/lockable/lockable.py#L19. 
basically those should be the same to be optimal. propose to use `DEFAULT_TIMEOUT` directly from lockable. Please double check these... :)

Reply:
pytest-lockable actually sets allocation_timeout to 10 by default so it never even reaches the default value.

If we want to use `DEFAULT_TIMEOUT` from lockable we would have to override it.